common/persistence/src/main/resources/derivation-maps.yaml

```diff
@@ -0,0 +1,138 @@
+# This externalised configuration is used for deriving missing values of Temporary Exposure Key fields
+# in modules that represent diagnosis key input channels.
+tek-field-derivations:
+  # As of version 1.5, EFGS can send DSOS values for which there is no TRL mapped, thus 
+  # we need a system configurable TRL value
```

Comment:
I'm confused, since the description talks about a "default dsos", but the variable is named "default trl"?

Reply:
@mibrasap   where is the default dsos mentioned?  It states that efgs can send dsos values for which we cannot derive TRL, thus we use the default trl.